mypy/plugins/dataclasses.py

```diff
@@ -20,10 +20,15 @@
 from mypy.server.trigger import make_wildcard_trigger
 
 # The set of decorators that generate dataclasses.
-dataclass_makers: Final = {
+dataclass_makers = {
```

Comment:
Why do you want to remove `Final` here?

Reply:
Just because I am editing it like so: https://github.com/NeilGirdhar/tjax/blob/master/tjax/mypy_plugin.py

But I'm happy to revert that change if it makes you uncomfortable.

Reply:
Oh I see 🙂 

There's no need to remove `Final` when mutating a variable. Only reassignment is forbidden with `Final`.
So, `mypy.plugins.dataclasses.dataclass_makers.add('tjax._src.dataclasses.dataclass.dataclass')` is fine. And `mypy.plugins.dataclasses.dataclass_makers = []` not.

Reply:
Ah!  Thanks for setting me straight on that.  I will re-add `Final` now.